app/src/main/java/com/example/mvptask/view/adapter/ImageAndDateLoader.java

```diff
@@ -0,0 +1,20 @@
+package com.example.mvptask.view.adapter;
```

Comment:
There should be a separate package called helpers, we should move all helper classes inside it including the constants and utilities classes